src/menu.c

```diff
@@ -693,8 +691,8 @@ static  int get_dipsw_b2(void)
 }
 static  void    cb_dipsw_b2(UNUSED_WIDGET, void *p)
 {
-    if ((int)p) reset_req.boot_from_rom = TRUE;
-    else        reset_req.boot_from_rom = FALSE;
+    if ((intptr_t)p)  reset_req.boot_from_rom = TRUE;
+    else              reset_req.boot_from_rom = FALSE;
```

Comment:
```suggestion
    else             reset_req.boot_from_rom = FALSE;
```
For realignment with the previous line.